operator/data/profiles/default.yaml

```diff
@@ -648,13 +653,13 @@ spec:
         accessMode: ReadWriteMany
       zipkin:
         hub: docker.io/openzipkin
-        tag: 2.14.2
-        probeStartupDelay: 200
+        tag: 2.20.0
```

Comment:
This was not intended for 1.5 backport

Reply:
do you mean just the tag or the whole block? i've changed only the tag back.

Reply:
everything (except the gateway part)

Reply:
all of this zipkin stuff though

Reply:
done